src/Cmp/Queues/Infrastructure/AmqpLib/v26/RabbitMQ/Queue/MessageHandler.php

```diff
@@ -32,7 +32,9 @@ public function __construct(JSONMessageFactory $jsonMessageFactory)
     public function handleMessage(AMQPMessage $message)
     {
         $task = $this->jsonMessageFactory->create($message->body);
-        call_user_func_array($this->callback, [$task]);
+        if (isset($this->callback)) {
```

Comment:
When the callback is not going to be set? And if that happens we want to ack the message?